modules/ob-enabling-builds-in-a-network-restricted-environment-using-the-source-to-image-build.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * builds/uninstalling-openshift-builds.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id='ob-creating-source-to-image-build-in-a-network-restricted-environment_{context}']
+= Creating source-to-image build in a network restricted environment
+
+You can create a `source-to-image` build in a network restricted environment by mirroring the images required by `source-to-image` BuildStrategy. 
+
+.Prerequisites
+
+* Your cluster can connect and interact with the git source that you will use to create the source-to-image build.
```

Comment:
```suggestion
* Your cluster can connect and interact with the git source that you can use to create the source-to-image build.
```

Reply:
IBMSG says to write in the present tense as much as possible: https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#tense